dev-resources/input/simple.json

```diff
@@ -1 +1 @@
w3id.org/xapi/cmi5#failedthenmaybecompleted"]},{"id":"https://w3id.org/xapi/cmi5#maybecompletedthenfailed","type":"Pattern","prefLabel":{"en":"Maybe Completed Then Failed"},"inScheme":"https://w3id.org/xapi/cmi5/v1.0","sequence":["https://w3id.org/xapi/cmi5#maybecompleted","https://w3id.org/xapi/cmi5#satisfieds","https://w3id.org/xapi/cmi5#failed"]},{"id":"https://w3id.org/xapi/cmi5#failedthenmaybecompleted","type":"Pattern","prefLabel":{"en":"Failed Then Maybe Completed"},"inScheme":"https://w3id.org/xapi/cmi5/v1.0","sequence":["https://w3id.org/xapi/cmi5#failed","https://w3id.org/xapi/cmi5#maybecompleted"]},{"id":"https://w3id.org/xapi/cmi5#maybecompleted","type":"Pattern","prefLabel":{"en":"Maybe Completed"},"inScheme":"https://w3id.org/xapi/cmi5/v1.0","optional":"https://w3id.org/xapi/cmi5#completed"},{"id":"https://w3id.org/xapi/cmi5#typicalsession","type":"Pattern","prefLabel":{"en":"Typical Session"},"inScheme":"https://w3id.org/xapi/cmi5/v1.0","alternates":["https://w3id.org/xapi/cmi5#completionmaybefailedsession","https://w3id.org/xapi/cmi5#completionpassedsession","https://w3id.org/xapi/cmi5#failedsession","https://w3id.org/xapi/cmi5#noresultsession","https://w3id.org/xapi/cmi5#passedsession","https://w3id.org/xapi/cmi5#completionnosuccesssession","https://w3id.org/xapi/cmi5#waivedsession"]},{"id":"https://w3id.org/xapi/cmi5#typicalsessions","type":"Pattern","prefLabel":{"en":"Typical Sessions"},"definition":{"en":"This pattern provides an option to use zero or more of the alernates for the Typical Session pattern."},"inScheme":"https://w3id.org/xapi/cmi5/v1.0","oneOrMore":"https://w3id.org/xapi/cmi5#typicalsession"},{"id":"https://w3id.org/xapi/cmi5#toplevel","type":"Pattern","primary":true,"prefLabel":{"en":"General Pattern"},"definition":{"en":"This pattern describes the sequence of statements sent over the an entire course registration."},"inScheme":"https://w3id.org/xapi/cmi5/v1.0","sequence":["https://w3id.org/xapi/cmi5#satisfieds","https://w3id.org/xapi/cmi5#typicalsessions"]}],"conformsTo":"https://w3id.org/xapi/profiles#1.0","prefLabel":{"en":"cmi5 Profile"}}],"personae":{"member":[{"name":"Bob Fakename","mbox":"mailto:bob@example.org","role":"Lead Developer"},{"name":"Alice Faux","mbox":"mailto:alice@example.org","role":"Data Engineer"},{"name":"Fred Ersatz","mbox":"mailto:fred@example.org"}],"objectType":"Group","name":"trainees"},"alignments":[{"id":"mbox::mailto:bob@example.org","type":"Agent","alignments":[{"component":"https://example.org/activity/a","weight":0.5,"objectOverride":{"objectType":"Activity","id":"https://www.whatever.com/activities#course2","definition":{"name":{"en-US":"Course 2"},"description":{"en-US":"Course Description 2"},"type":"http://adlnet.gov/expapi/activities/course"}}},{"component":"https://example.org/activity/c","weight":-0.2}]}],"parameters":{"start":"2019-11-18T11:38:39.219768Z","end":"2019-11-19T11:38:39.219768Z","timezone":"America/New_York","seed":42,"from":"2019-11-18T11:38:39.219768Z"}}
```

Comment:
Since we don't pprint this file (maybe we should), wanna just let me know what changed?

Reply:
Oh, I assume this is just the compiled version of the below changes, nm

Reply:
I do want to confirm that we didn't put any overrides into the `simple.json` input file. It would be good to make an example input file to show the functionality though

Reply:
Oh I did put overrides, since I added them to `alignments/simple.json`. I actually originally had them in a separate `alignments/simple_overrides.json` file, so i can restore it if that is what you prefer.

Reply:
Yeah, I think because it's a feature kind of outside the norm it would be good to show it its own json file and leave the simple stuff simple

Reply:
Roles can stay in `simple.json` though, right?

Reply:
yep!